api/interview_routes.py

```
@@ -334,9 +342,21 @@ async def start_technical_interview(request: StartTechnicalRequest):
         answers = session.interview.get_answers()
         session.general_analysis = analyze_general_interview(answers)
 
+    # 백엔드 API에서 프로필 가져오기
+    backend_client = get_backend_client()
+    try:
+        # TODO: user_id는 세션에서 가져오거나 JWT에서 파싱
+        user_id = 1  # 임시값
+        profile = await backend_client.get_talent_profile(user_id, request.access_token)
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)